charts/minecraft-bedrock/ci/test-values.yaml

```diff
@@ -0,0 +1,3 @@
+# This must be overridden, since we can't accept this for the user.
+minecraftServer.eula: "TRUE"
```

Comment:
This needs to be,

```yaml
minecraftServer:
  eula: "TRUE"
```

For the test helm install to consume it properly.  The existing minecraft chart needs to be fixed as well.

Reply:
I thought that was done intentionally to prevent people from blindly accepting the EULA?

Reply:
I’m confused too @billimek. I’d prefer to fix the testing action rather than have the chart come pre-baked with the eula accepted.

Reply:
...oh, wait that’s the test-values. That seems fine then.

Reply:
Yeah, this is only present for automated tests and is never used by real users at runtime.